mathics/builtin/compilation.py

```diff
@@ -161,6 +161,9 @@ def __init__(self, cfunc, args, **kwargs):
         self.cfunc = cfunc
         self.args = args
 
+    def equal2(self, rhs):
```

Comment:
Thanks! This is what I had been intending to add at some point.

Reply:
Not sure if the content of this method is what we want at the end, but this works by now.